test/components/rankers/test_transformers_similarity.py

```diff
@@ -421,20 +423,21 @@ def test_run_small_batch_size(self, query, docs_before_texts, expected_first_tex
 
     #  Returns an empty list if no documents are provided
     @pytest.mark.integration
+    @pytest.mark.slow
     def test_returns_empty_list_if_no_documents_are_provided(self):
         sampler = TransformersSimilarityRanker()
         sampler.warm_up()
         output = sampler.run(query="City in Germany", documents=[])
         assert not output["documents"]
 
     #  Raises ComponentError if model is not warmed up
-    @pytest.mark.integration
     def test_raises_component_error_if_model_not_warmed_up(self):
         sampler = TransformersSimilarityRanker()
         with pytest.raises(RuntimeError):
             sampler.run(query="query", documents=[Document(content="document")])
 
     @pytest.mark.integration
+    @pytest.mark.slow
     @pytest.mark.parametrize(
         "query,docs_before_texts,expected_first_text",
         [
```

Comment:
To slightly help with speed I don't think there is any reason we should be parametrizing the tests in this file. Doing just one is enough to test the behavior. We don't need to run three which should also help save on cpu inference time.

Reply:
done in https://github.com/deepset-ai/haystack/pull/9296/commits/a31d1574eeede2ea202b44b66e65f09af454e4f9